### PR TITLE
Event email

### PR DESCRIPTION
**Pull request by lramos15**
Allows the easy sending of event emails for RPIA events.

### CODE REVIEW

File: .mailer.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+if (getUser($_POST['sessionId'], $connection)['admin'] == 0) {
+    $data['success'] = false;
+    echo (json_encode($data));
+    $connection = null;
+    return;
+}
+
+$connection = null;
+
+$url = 'http://lp13.rpiambulance.com:3000/sendmail';
```
Comment (ddbruce) [inline .mailer.php]: This needs to be something like `mailer.lp13.rpiambulance.com`

Comment (ddbruce) [reply .mailer.php]: And probably should be `https`

File: js/controllers/GameCtrl.js [hunk 2]
```diff
@@ -28,105 +28,172 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', '$http', '$locati
...
+    $scope.displayModal = () => {
+        $scope.showModal = true;
+    }
+
+    $scope.sendEmail = () => {
+        $scope.showModal = false;
+        if (!$scope.admin) {
+            return;
+        }
+        let body = `Dear Attendees,\n\nThank you for signing up for ${$scope.game.description} with RPI Ambulance. The event is ${$scope.game.date} at ${$scope.game.start} hours. Please meet at the garage 5 minutes prior to ensure a prompt departure.\n\n`;
```
Comment (ddbruce) [inline js/controllers/GameCtrl.js]: Please remove "Dear Attendees"

File: js/controllers/GameCtrl.js [hunk 3]
```diff
@@ -28,105 +28,172 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', '$http', '$locati
...
+        body += 'Attendance is as follows:\n';
+        $scope.positions.forEach((p) => {
+            if (p.attendees.length != 0) {
+                body += `${p.title}[s]: ${p.attendees[0].ambulance_name}`;
```
Comment (ddbruce) [inline js/controllers/GameCtrl.js]: Please use a ternary or something like:

```
${p.title == 1 ? p.title : p.title + "s"}:
```  

I think this is valid code.

File: js/controllers/GameCtrl.js [hunk 4]
```diff
@@ -28,105 +28,172 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', '$http', '$locati
...
+                for (let i = 1; i < p.attendees.length; i++) {
+                    body += `, ${p.attendees[0].ambulance_name}`;
+                }
+                body += '\n';
+            }
+        });
+        body += `\nAdditional Info:\n${$scope.formData.additionalinfo}`;
+        body += '\n If you have any questions, you can reach out to secondlt@rpiambualnce.com. Thanks!\n\n';
```
Comment (ddbruce) [inline js/controllers/GameCtrl.js]: Or who sent this email (if it wasn't the 2nd lt)

Comment (lramos15) [reply js/controllers/GameCtrl.js]: Only the 2nd lt should be sending event emails.

File: .mailer.php (hunk 1 above)
Comment (lramos15) [reply .mailer.php]: Updated

File: js/controllers/GameCtrl.js (hunk 2 above)
Comment (lramos15) [reply js/controllers/GameCtrl.js]: Fixed.

File: js/controllers/GameCtrl.js (hunk 3 above)
Comment (lramos15) [reply js/controllers/GameCtrl.js]: Fixed

File: js/controllers/GameCtrl.js (hunk 4 above)
Comment (ddbruce) [reply js/controllers/GameCtrl.js]: Good point. I do think we should add the EES info in there if it's a EESed event. They're a good POC

Comment (lramos15) [reply js/controllers/GameCtrl.js]: Per Car 3. I should just include 2nd lt and he will route them accordingly

Comment (ddbruce) [reply js/controllers/GameCtrl.js]: I never understood why people choose to create more work for themselves, but ok.

**Pull request merged**